test/OpenTelemetry.Instrumentation.EventCounters.Tests/EventCountersMetricsTests.cs

```diff
@@ -104,7 +104,7 @@ public async Task IncrementingEventCounter()
         Assert.Equal(3, GetActualValue(metric));
     }
 
-    [Fact]
+    [Fact(Skip = "Unstable")]
```

Comment:
I think that it will be good idea to create separate issue for each skipped tests/class contains more than one skipped tests and link it from source code.